### PR TITLE
Release Godot 4.3 dev 3 and publish release notes

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-4-3-dev-3.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+### Single-threaded web exports
+
+For Godot 4.0, we modernized the engine to make heavier use of multi-threading. The web was the last platform where multi-threading wasn't a given, but support for the required [SharedArrayBuffer](https://caniuse.com/sharedarraybuffer) feature finally seemed widespread (reaching Safari at last), so we decided to go all in and also make Godot's Web export multi-threaded by default, solving a number of audio issues we had in Godot 3.
+
+Experience has proven that even though SharedArrayBuffer *is* supported by all browsers nowaday, the condition it imposes on the web server that host the games are too difficult to uphold. For people who self-host, it's easy enough, but for people who distribute their games on publishing platforms like itch.io or CrazyGames, it's often outside their control. The requirements for SharedArrayBuffer (for security reasons) are also at odds with web game monetization options, such as advertisement or payment processing.
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-dev-3.md]:
```suggestion
Experience has proven that even though SharedArrayBuffer *is* supported by all browsers nowadays, the conditions it imposes on the web server that host the games are too difficult to uphold. For people who self-host, it's easy enough, but for people who distribute their games on publishing platforms like itch.io or CrazyGames, it's often outside their control. The requirements for SharedArrayBuffer (for security reasons) are also at odds with web game monetization options, such as advertisement or payment processing.
```
Alternatively "conditions ... are".

File: collections/_article/dev-snapshot-godot-4-3-dev-3.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+The implementation we merged was a massive undertaking led by Riteo ([GH-86180](https://github.com/godotengine/godot/pull/86180)), spanning 2 years of development with extensive testing and contributions by many others. This built upon previous attempts ([GH-23426](https://github.com/godotengine/godot/pull/23426), [GH-27463](https://github.com/godotengine/godot/pull/27463)) which came at a time where Godot's architecture wasn't ready for it yet, notably before the 4.0 split between OS (platform) and DisplayServer responsibilities.
+
+As with everything introduced in these dev snapshots, this is a **work in progress** and the early days of Wayland support in the main branch, which we will iterate upon in coming months to ensure that we eventually have feature parity with the X11 backend, which is still the default currently.
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-dev-3.md]:
```suggestion
As with everything introduced in these dev snapshots, this is a **work in progress** and the early days of Wayland support in the main branch. We will iterate upon this work in coming months to ensure that we eventually have feature parity with the X11 backend, which is still the default currently.
```
Broke up long sentence.

File: collections/_article/dev-snapshot-godot-4-3-dev-3.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+In this build however, we enabled Direct3D 12 support in our official builds, including the open source Mesa NIR library and Godot's D3D12 implementation. `dxil.dll` is still required, but not provided, so by default you will still only have access to the Vulkan backend. To enable the D3D12 support, you need to download the [DirectX Shader Compiler](https://github.com/Microsoft/DirectXShaderCompiler/releases), and copy the relevant `dxil.dll` file for your architecture next to your Godot editor or exported project's executable.
+
+We are still evaluating options for being able to provide a D3D12 support that works out of the box, without proprietary component. But for now to test things you will have to do this manual step (or [compile from source](https://docs.godotengine.org/en/latest/contributing/development/compiling/compiling_for_windows.html#compiling-with-support-for-direct3d-12), which does it for you).
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-dev-3.md]:
```suggestion
We are still evaluating options for being able to provide a D3D12 support that works out of the box, without a proprietary component. But for now to test things you will have to do this manual step (or [compile from source](https://docs.godotengine.org/en/latest/contributing/development/compiling/compiling_for_windows.html#compiling-with-support-for-direct3d-12), which does it for you).
```

**Review by coppolaemilio — APPROVED** [review]

**Review by deralmas — APPROVED** [review]
Looks good, especially the Wayland part!

**Review by adamscott — APPROVED** [review]
Amazing article. Thanks Rémi!

File: collections/_article/dev-snapshot-godot-4-3-dev-3.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+- GraphEdit connections have been reworked to improve their drawing and API, and optimize them ([GH-86158](https://github.com/godotengine/godot/pull/86158)).
+
+### And more!
+
+The above section is already too long for "just" a dev snapshot, but that's how it grows with a month's worth of development. Here are some other significant changes you might be interested in:
+
+- Add colors to the command-line help ([GH-36252](https://github.com/godotengine/godot/pull/36252)).
+- Add option to reverse FlowContainer fill direction (HFlow bottom-to-top, VFlow right-to-left) ([GH-74195](https://github.com/godotengine/godot/pull/74195)).
+- Support detecting and mapping Ctrl/Alt/Shift/Meta by their left/right physical location ([GH-80231](https://github.com/godotengine/godot/pull/80231)).
+- New VS project generation ([GH-84885](https://github.com/godotengine/godot/pull/84885)).
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-dev-3.md]:
```suggestion
- New Visual Studio project generation ([GH-84885](https://github.com/godotengine/godot/pull/84885)).
```

File: collections/_article/dev-snapshot-godot-4-3-dev-3.md (hunk 2 above)
Comment (akien-mga) [reply collections/_article/dev-snapshot-godot-4-3-dev-3.md]: Thanks a lot, I had the feeling I wrote some pretty heavy sentences in this blog post :D

**Comment by coppolaemilio** [comment]
🚀

**Pull request merged**